glean-core/python/setup.py

```diff
@@ -126,6 +118,34 @@ def finalize_options(self):
             self.install_lib = self.install_platlib
 
 
+class build(_build):
+    def run(self):
+        try:
+            subprocess.run(["cargo"])
+        except subprocess.CalledProcessError:
+            print("glean_sdk requires 'cargo' to build its Rust extension.")
```

Comment:
Maybe expand this, e.g.

"Install Rust and Cargo through Rustup: https://rustup.rs/. Need help with installing the glean_sdk? Contact the Glean Team: ..."